code/game/objects/items/tools/wirecutters.dm

```diff
@@ -145,7 +148,10 @@
 
 	return OXYLOSS
 
-/obj/item/wirecutters/power/attack_self__legacy__attackchain(mob/user)
+/obj/item/wirecutters/power/activate_self(mob/user)
+	if(..())
```

Comment:
Did you test and make sure jaws could still be swapped back and forth?

Reply:
Uhhh I don't remember. I'll check the moment I'm home

Reply:
Yup, switches them effortlessly!